Bottom: b5b0018b79bc03e6a1efc99625c16c9897813165
Top:    fd1e116a468a98ac175552ba6e3a7823a99bc932
Author: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>
Date:   2022-12-28 21:02:01 +0000

RDEPENDS skip e.g. normalize-resolvconf with busybox

Signed-off-by: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>


---

diff --git a/meta/recipes-connectivity/resolvconf/resolvconf_1.91.bb b/meta/recipes-connectivity/resolvconf/resolvconf_1.91.bb
index 3f1b75d07d..be5a3e5836 100644
--- a/meta/recipes-connectivity/resolvconf/resolvconf_1.91.bb
+++ b/meta/recipes-connectivity/resolvconf/resolvconf_1.91.bb
@@ -11,6 +11,12 @@ AUTHOR = "Thomas Hood"
 HOMEPAGE = "http://packages.debian.org/resolvconf"
 RDEPENDS:${PN} = "bash sed util-linux-flock"
 
+# The normalize-resolvconf sed script uses /bin/sed on the shebang line.
+# Ignore the file-rdeps test so when using busybox we don't get an RDEPENDS error.
+#
+# busybox: /bin/sed -> /bin/busybox.nosuid
+INSANE_SKIP:${PN} += "file-rdeps"
+
 SRC_URI = "git://salsa.debian.org/debian/resolvconf.git;protocol=https;branch=unstable \
            file://99_resolvconf \
            file://0001-avoid-using-m-option-for-readlink.patch \
